uniqueWordAbbreviation.py

```diff
@@ -5,8 +5,8 @@ def __init__(self, dictionary):
         :type dictionary: List[str]
         """
         self.summary = {}
```

Comment:
Why do you need `self.summary`?

Reply:
Because the function needs initialisation.